spark/src/test/scala/org/apache/comet/CometExpressionSuite.scala

```diff
@@ -850,6 +850,34 @@ class CometExpressionSuite extends CometTestBase with AdaptiveSparkPlanHelper {
     }
   }
 
+  test("abs Overflow ansi mode") {
+    val data: Seq[(Int, Int)] = Seq((Int.MaxValue, Int.MinValue))
```

Comment:
Can we have tests for all numerical values?

Reply:
done!